feature/search/presentation/src/main/java/com/sanaa/presentation/screen/componants/SearchSection.kt

```diff
@@ -0,0 +1,54 @@
+package com.sanaa.presentation.screen.componants
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sanaa.designsystem.R
+import com.sanaa.designsystem.design_system.component.text_field.NovixTextField
+
+
+@Composable
+fun SearchSection(
+    onClick: ()->Unit = {}
+) {
+   Row(
+       modifier = Modifier
+           .height(56.dp)
+           .padding(horizontal = 16.dp)
+           .fillMaxWidth(),
+       horizontalArrangement = Arrangement.spacedBy(space = 8.dp),
+       verticalAlignment = Alignment.CenterVertically
+   ) {
+       var text by remember { mutableStateOf("") }
```

Comment:
no need for this variable just pass the text to SearchSection
@Composable
fun SearchSection(
    searchQuery: String,
    onClick: ()->Unit = {},
) {
}